crates/primitives/state/Cargo.toml

```diff
@@ -12,8 +12,34 @@ targets = ["x86_64-unknown-linux-gnu"]
 
 [dependencies]
 blockifier = { workspace = true }
+mp-felt = { workspace = true, features = [
+  "parity-scale-codec",
+  "serde",
+  "scale-info",
```

Comment:
same, they go bellow, in the features

Reply:
Correct me if I'm wrong but seems that I only forgot for scale-info here

Reply:
Same as above, you can remove ` features = [
  "parity-scale-codec",
  "serde",
  "scale-info",
  ]` as it is managed below in the `features` section